docs/section5/Rule5-37.md

```diff
@@ -17,68 +17,66 @@
+  3. get_rmr_scc_skylight_roof_ratios()
   4. data_lookup()
 
 ## Rule Logic:  
 
-- Get building climate zone: `climate_zone = B_RMR.weather.climate_zone`  
+- Get RMR climate zone: `climate_zone = ASHRAE229.weather.climate_zone`  
 
 - Get surface conditioning category dictionary for B_RMR: `scc_dictionary_b = get_surface_conditioning_category(B_RMR)`  
 
-- Get building skylight roof ratios dictionary: `building_scc_skylight_roof_ratios_dictionary = get_building_scc_skylight_roof_ratios(B_RMR)`
+- Get B_RMR skylight roof ratios dictionary: `rmr_scc_skylight_roof_ratios_dictionary = get_rmr_scc_skylight_roof_ratios(B_RMR)`
 
-- For each building in the Baseline model: `for building_b in B_RMR.building:`
+- Check if B_RMR has exterior mixed type skylight: `if rmr_scc_skylight_roof_ratios_dictionary["EXTERIOR MIXED"] > 0:`
 
-  - Check if building has exterior mixed type skylight: `if building_scc_skylight_roof_ratios_dictionary[building_b.id]["EXTERIOR MIXED"] > 0:`
+  - Check if residential and non-residential type skylight U-factor requirements for different skylight-roof-ratio are the same, get skylight U-factor requirements: `if ( data_lookup(table_G3_4, climate_zone, "RESIDENTIAL, "SKYLIGHT", "0%-2.0%", "ASSEMBLY MAX. U") == data_lookup(table_G3_4, climate_zone, "RESIDENTIAL, "SKYLIGHT", "2.1%+", "ASSEMBLY MAX. U") ) AND ( data_lookup(table_G3_4, climate_zone, "NON-RESIDENTIAL, "SKYLIGHT", "0%-2.0%", "ASSEMBLY MAX. U") == data_lookup(table_G3_4, climate_zone, "NON-RESIDENTIAL, "SKYLIGHT", "2.1%+", "ASSEMBLY MAX. U") ) AND ( data_lookup(table_G3_4, climate_zone, "RESIDENTIAL, "SKYLIGHT", "0%-2.0%", "ASSEMBLY MAX. U") == data_lookup(table_G3_4, climate_zone, "NON-RESIDENTIAL, "SKYLIGHT", "0%-2.0%", "ASSEMBLY MAX. U") ): target_u_factor_mixed = data_lookup(table_G3_4, climate_zone, "RESIDENTIAL, "SKYLIGHT", "0%-2.0%", "ASSEMBLY MAX. U")`
 
-    - Check if residential and non-residential type skylight U-factor requirements for different skylight-roof-ratio are the same, get skylight U-factor requirements: `if ( data_lookup(table_G3_4, climate_zone, "RESIDENTIAL, "SKYLIGHT", "0%-2.0%", "ASSEMBLY MAX. U") == data_lookup(table_G3_4, climate_zone, "RESIDENTIAL, "SKYLIGHT", "2.1%+", "ASSEMBLY MAX. U") ) AND ( data_lookup(table_G3_4, climate_zone, "NON-RESIDENTIAL, "SKYLIGHT", "0%-2.0%", "ASSEMBLY MAX. U") == data_lookup(table_G3_4, climate_zone, "NON-RESIDENTIAL, "SKYLIGHT", "2.1%+", "ASSEMBLY MAX. U") ) AND ( data_lookup(table_G3_4, climate_zone, "RESIDENTIAL, "SKYLIGHT", "0%-2.0%", "ASSEMBLY MAX. U") == data_lookup(table_G3_4, climate_zone, "NON-RESIDENTIAL, "SKYLIGHT", "0%-2.0%", "ASSEMBLY MAX. U") ): target_u_factor_mixed = data_lookup(table_G3_4, climate_zone, "RESIDENTIAL, "SKYLIGHT", "0%-2.0%", "ASSEMBLY MAX. U")`
+  - Else, request manual review: `else: manual_review_flag = TRUE`
 
-    - Else, request manual review: `else: manual_review_flag = TRUE`
+- Else, B_RMR does not have exterior mixed type roof surface: `else:`
```

Comment:
Should line 37 be removed?

Reply:
This "else" is for baseline that does not have exterior mixed type skylight.